.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
I think we don't need to force any specific version.

Reply:
I will remove, I had to add it locally to get things to work as running `bundle` locally with Ruby 3.3.0 was giving me an error with `File.exists?` being removed in Ruby 3.2.0

Reply:
Addressed in ca12c202133d6e39590fe4a918490f72f6a600e7